### PR TITLE
feat/turoials

### PR DESCRIPTION
**Pull request by Ives7**

### CODE REVIEW

File: docs/tutorials.md [hunk 1]
```diff
@@ -10,38 +10,335 @@ shuvi is a front-end application development solution that integrates modern too
...
+- use page route
+- data fetching
+- use dynamic route
+- use layout route
+- use css modules
+- how to deploy
+
 ### System Requirements
 
 - [node ^14.x](https://github.com/nodejs/Release)
```
Comment (liximomo) [inline docs/tutorials.md]: 16.x

**Comment by Ives7** [comment]
image

**Comment by Ives7** [comment]
image

**Comment by Ives7** [comment]
image

**Comment by Ives7** [comment]
image
image
image
image
image
image

image
image

File: docs/tutorials.md [hunk 2]
```diff
@@ -10,38 +10,335 @@ shuvi is a front-end application development solution that integrates modern too
 
 ## Blog
```
Comment (liximomo) [inline docs/tutorials.md]: 改成 Create your first app, 直接叫blog我还以为后面的内容是我们的博客呢

**Review by liximomo — CHANGES_REQUESTED** [review]

**Comment by Ives7** [comment]
finnal /posts

image

**Comment by Ives7** [comment]
finnal /posts/:id
image

**Comment by Ives7** [comment]
image

**Comment by Ives7** [comment]
image

**Pull request merged**